content/docs/studio/how-it-works.md

```diff
@@ -1,10 +1,8 @@
 # How Iterative Studio Works
 
 [Iterative Studio](https://studio.iterative.ai/) works very closely with your
-Git ecosystem. It parses the metadata found in your ML project repositories to
-present organized data, hyperparameters, models, and metrics on a web UI. It
-also parses and displays live metrics and plots sent by [DVCLive]; these don't
-need to be pushed to the Git repositories.
+Git ecosystem. It parses the metadata from your ML project to present organized
+data, hyperparameters, models, and metrics on a web UI.
```

Comment:
> Rel. https://github.com/iterative/dvc.org/pull/4012#pullrequestreview-1218407058

Can we just use more general language to avoid being so specific at the very top here? DVCLive is mentioned later again anyway (in a clearer way).

Reply:
The project metadata in the repository does NOT have live metrics and plots.

Reply:
OK I still think we could be more general but this is minor. Resolving